default_weights.toml

```diff
@@ -0,0 +1,119 @@
+[base]
+perm_temp_for_expr = 100
+perm_expand_expr = 20
+perm_reorder_stmts = 20
+perm_add_mask = 15
+perm_xor_zero = 10
+perm_cast_simple = 10
+perm_refer_to_var = 10
+perm_float_literal = 10
+perm_randomize_internal_type = 10
+perm_randomize_external_type = 5
+perm_randomize_function_type = 5
+perm_split_assignment = 10
+perm_sameline = 10
+perm_ins_block = 10
+perm_struct_ref = 10
+perm_empty_stmt = 10
+perm_condition = 10
+perm_mult_zero = 5
+perm_dummy_comma_expr = 5
+perm_add_self_assignment = 5
+perm_commutative = 5
+perm_add_sub = 5
+perm_inequalities = 5
+perm_compound_assignment = 5
+perm_remove_ast = 5
+perm_duplicate_assignment = 5
+perm_chain_assignment = 5
+perm_pad_var_decl = 1
+
+[ido]
```

Comment:
These should extend base, don't include the values that are the same.

If you want some non-default weights to start out with, `float_literal` and `sameline` could be reduced (10 -> 3?) for non-IDO

Reply:
done